src/getCoinCombination.test.js

```diff
@@ -7,4 +7,44 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return [0, 0, 0, 0] for 0 ', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
+
+  it('If accept pennys only', () => {
```

Comment:
The test description 'If accept pennys only' should be revised for clarity. Consider changing it to 'should return [1, 0, 0, 0] for 1 cent (only pennies)'. Also, note that the correct spelling is 'pennies'.